### PR TITLE
AndBeyondMedia Adapter: add userSync

### PR DESCRIPTION
**Pull request by AndBeyondMediaHB**

### CODE REVIEW

File: static/bidder-info/beyondmedia.yaml [hunk 1]
```diff
@@ -13,3 +13,7 @@ capabilities:
     - banner
     - video
     - native
+userSync:
+  redirect:
+    url: "https://cookies.andbeyond.media/pbserver?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redir={{.RedirectURL}}"
```
[the remark this answers is not in the document]
Comment (SyntaxNode) [reply static/bidder-info/beyondmedia.yaml]: Same here. I cannot establish a connection to `cookies.andbeyond.media`.

[the remark this answers is not in the document]
Comment (hhhjort) [reply static/bidder-info/beyondmedia.yaml]: Fixing the query string parameters doesn't help if the server doesn't allow any connections.

[the remark this answers is not in the document]
Comment (AndBeyondMediaHB) [reply static/bidder-info/beyondmedia.yaml]: @hhhjort @SyntaxNode thx, we fixed issue. Working link - https://cookies.andbeyond.media/pbserver?gdpr=1&gdpr_consent=testconsent&ccpa=testccpa&redir=https%3A%2F%2Fexample.com

[the remark this answers is not in the document]
Comment (hhhjort) [reply static/bidder-info/beyondmedia.yaml]: There does seem to be an issue when using `gdpr=&gdpr_consent=`, but it does work with `gdpr=0`. You probably want to ensure that your cookie sync endpoint works when the `gdpr` flag is not set, as not all people outside the scope of the GDPR will bother with setting the flag.

[the remark this answers is not in the document]
Comment (onkarvhanumante) [reply static/bidder-info/beyondmedia.yaml]: @AndBeyondMediaHB  thank you for fixing redirect url

Following url worked for me:

https://cookies.andbeyond.media/pbserver?gdpr=&gdpr_consent=&ccpa=&redir=https%3A%2F%2Fprebid.adnxs.com%2Fpbs%2Fv1%2Fsetuid%3Fbidder%3Dbeyondmedia%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%5BUID%5D


https://cookies.andbeyond.media/pbserver?gdpr=0&gdpr_consent=&ccpa=&redir=https%3A%2F%2Fprebid.adnxs.com%2Fpbs%2Fv1%2Fsetuid%3Fbidder%3Dbeyondmedia%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%5BUID%5D

**Review by onkarvhanumante — APPROVED** [review]

File: static/bidder-info/beyondmedia.yaml [hunk 2]
```diff
@@ -13,3 +13,7 @@ capabilities:
     - banner
     - video
     - native
+userSync:
+  redirect:
+    url: "https://cookies.andbeyond.media/pbserver?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&ccpa={{.USPrivacy}}&redir={{.RedirectURL}}"
```
Comment (Sonali-More-Xandr) [inline static/bidder-info/beyondmedia.yaml:18]: LGTM! Redirect is working for me.

**Pull request merged**